bin/twister.es6

```diff
@@ -0,0 +1,117 @@
+#!/usr/bin/env node
+
```

Comment:
IMO, there's an expectation this'll be named `bin/tornado` or `bin/tornado-cli`  or `bin/cli`